### PR TITLE
Add description to homepage

### PR DESCRIPTION
**Pull request by vadi2**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ layout: page
 
 ## Shutter website
 
-Something meaningful should be written here.
\ No newline at end of file
+Shutter is a feature-rich screenshot program for Linux based operating systems such as Ubuntu. You can take a screenshot of a specific area, window, your whole screen, or even of a website – apply different effects to it, draw on it to highlight points, and then upload to an image hosting site, all within one window. Shutter is free, [open-source](http://www.opensource.org/docs/definition.php), and licensed under [GPL v3](http://www.gnu.org/licenses/gpl-3.0.html).
```
Comment (DarthGandalf) [inline index.md]: Is anything stopping it from working on non-linux? like freebsd

Comment (DarthGandalf) [inline index.md]: https:// please

Comment (vadi2) [reply index.md]: Not sure, but the focus is on Linux primarily. Happy if others can run it too.

Comment (vadi2) [inline index.md]:
```suggestion
Shutter is a feature-rich screenshot program for Linux based operating systems such as Ubuntu. You can take a screenshot of a specific area, window, your whole screen, or even of a website – apply different effects to it, draw on it to highlight points, and then upload to an image hosting site, all within one window. Shutter is free, [open-source](https://www.opensource.org/docs/definition.php), and licensed under [GPL v3](https://www.gnu.org/licenses/gpl-3.0.html).
```

**Review by Photon89 — APPROVED** [review]

**Pull request merged**